packages/xchain-thorchain-query/src/utils/thornode.ts

```diff
@@ -378,9 +393,9 @@ export class Thornode {
     toAsset: string,
     amount: number,
     destination: string,
-    toleranceBps: number,
+    toleranceBps: string,
     affiliateBps: number,
-    affiliate: string,
+    affiliate: number,
     height?: number,
```

Comment:
pretty sure these have not changed types, it looks like the api.quoteSwap() added a new optional param called fromAddress?: string,  this pososibly throws off the other params

(see attached screenshot)

![Screen Shot 2023-03-02 at 10 01 17 PM](https://user-images.githubusercontent.com/3811158/222635553-f434d3c3-6302-4299-b23c-972ad1d08aa9.png)

Reply:
good find, thanks